components/ActivityForm/index.js

```diff
@@ -112,16 +136,19 @@ export default function ActivityForm({ onAddActivity }) {
           cols="30"
           rows="10"
           placeholder="add description"
+          defaultValue={initialData?.description}
         ></StyledTextarea>
         <StyledLabel htmlFor="image">Image</StyledLabel>
         <StyledInput
           id="image"
           name="image"
           type="text"
           placeholder="https://example.com/image.jpg"
-          defaultValue={"/images/default-image.jpg"}
+          defaultValue={
+            isEditMode ? initialData?.image : "/images/default-image.jpg"
+          }
         />
-        <StyledButton type="submit">Add</StyledButton>
+        <StyledButton type="submit">{isEditMode ? "Save" : "Add"}</StyledButton>
```

Comment:
submit is the default type.

Reply:
should we delete the type=submit property here?